From 815c9e8da001a7d860b916c21d19668e2fcf0557 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Mon, 2 Nov 2015 21:48:11 +0100
Subject: [PATCH] rename 'sa' to 'adaBoostSegmentationRefinement'

/usr/sbin/sa provided by psacct which is well-known utility and should be renamed.

Reference: https://github.com/stnava/ANTs/issues/242
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 ExternalApplications/MALF/CMakeLists.txt | 6 +++---
 ExternalApplications/MALF/README.md      | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/ExternalApplications/MALF/CMakeLists.txt b/ExternalApplications/MALF/CMakeLists.txt
index 061a5fa..d6a93d4 100755
--- a/ExternalApplications/MALF/CMakeLists.txt
+++ b/ExternalApplications/MALF/CMakeLists.txt
@@ -8,15 +8,15 @@
 project(PICSL_MALF)
 
 add_executable(jointfusion ./JointFusion/LabelFusion.cxx)
-add_executable(sa ./CorrectiveLearning/segAdapter.cxx)
+add_executable(adaBoostSegmentationRefinement ./CorrectiveLearning/segAdapter.cxx)
 add_executable(bl ./CorrectiveLearning/BiasLearn.cxx)
 
 target_link_libraries(jointfusion ${ITK_LIBRARIES} )
-target_link_libraries(sa ${ITK_LIBRARIES} )
+target_link_libraries(adaBoostSegmentationRefinement ${ITK_LIBRARIES} )
 target_link_libraries(bl ${ITK_LIBRARIES} )
 
 
-install(TARGETS jointfusion sa bl
+install(TARGETS jointfusion adaBoostSegmentationRefinement bl
         RUNTIME DESTINATION bin
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
diff --git a/ExternalApplications/MALF/README.md b/ExternalApplications/MALF/README.md
index 7517640..d5d84aa 100644
--- a/ExternalApplications/MALF/README.md
+++ b/ExternalApplications/MALF/README.md
@@ -41,9 +41,9 @@ make
 to complie and build the executable files. Our program uses ITK's I/O functions to handle image input
 and output. Hence, it requries prebuilt ITK. The following three executable files will be built,
 
-jointfusion : joint label fusion
-bl          : learning classifiers for correcting systematic errors
-sa          : apply the learned classifiers to correct systematic segmentation errors on testing images
+jointfusion                    : joint label fusion
+bl                             : learning classifiers for correcting systematic errors
+adaBoostSegmentationRefinement : apply the learned classifiers to correct systematic segmentation errors on testing images
 
 Type each command to see details on how to use them.
 
-- 
2.6.2

